Gemfile

```
@@ -70,7 +70,7 @@ gem 'omniauth-lti', git: "https://github.com/avalonmediasystem/omniauth-lti.git"
 gem "omniauth-saml", "~> 2.0"
 
 # Media Access & Transcoding
-gem 'active_encode', '~> 1.0'
+gem 'active_encode', '~> 1.1.2'
```

This would restrict to only 1.1.x versions so I think this might be better:
```suggestion
gem 'active_encode', '~> 1.0', '>= 1.1.2'
```

---

Right. Good call. Forgot that was how the `~>` works.